packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -499,6 +499,7 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 			(error) => {
 				this.mc.logger.sendTelemetryEvent({
 					eventName: "UploadBlobReject",
+					// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
In `channelCollection.ts` we went for declaring `(error: Error)`, is that not possible here? I'd also be concerned about cases where we throw a string though.

Reply:
The typing here is complicated and a bit strange. `sendTelemetryEvent` would not accept `Error` nor `unkown`. I'll add a TODO to this one as well, but I don't want to spend too much time on it (my biggest priority is preventing new violations).

Reply:
```suggestion
					// TODO: better typing
					// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```